OpenImis.ePayment/Logic/PaymentLogic.cs

```diff
@@ -570,10 +570,10 @@ public List<BulkControlNumbersForEO> GetControlNumbersForEO(string officerCode,
             return imisPayment.GetControlNumbersForEO(officerCode, productCode);
         }
 
-        public int ControlNumbersToBeRequested(string productCode)
+        public async Task<int> ControlNumbersToBeRequested(string productCode)
         {
             var imisPayment = new ImisPayment(_configuration, _hostingEnvironment);
-            return imisPayment.ControlNumbersToBeRequested(productCode);
+            return await imisPayment.ControlNumbersToBeRequested(productCode);
```

Comment:
This line generates an error on compilation. Please check and fix.

Reply:
How? It compiles well on my machine. In fact the same version is deployed on the production. Tested and confirmed.